examples/llama/main.zig

```diff
@@ -1,156 +1,228 @@
 const asynk = @import("async");
-const flags = @import("tigerbeetle/flags");
+const clap = @import("clap");
 const std = @import("std");
 const stdx = @import("stdx");
 const zml = @import("zml");
 
-const llama_mod = @import("llama.zig");
+const llama = @import("llama.zig");
 
-const LlamaLM = llama_mod.LlamaLM;
-const Llama = llama_mod.Llama;
-const KvCache = llama_mod.KvCache;
-const TransformerLayer = llama_mod.TransformerLayer;
-const SelfAttn = llama_mod.SelfAttn;
+const LlamaLM = llama.LlamaLM;
+const Llama = llama.Llama;
+const KvCache = llama.KvCache;
+const TransformerLayer = llama.TransformerLayer;
+const SelfAttn = llama.SelfAttn;
 const Buffer = zml.Buffer;
 const Tensor = zml.Tensor;
 const ShapeOf = zml.ShapeOf;
 
 const log = std.log.scoped(.llama);
 
-const eos_tokens: [3]i32 = .{ 128001, 128008, 128009 };
-
 // set this to false to disable the verbose logging
-const show_mlir = true;
+const show_mlir = false;
 
 pub const std_options = .{
     .log_level = .warn,
     .log_scope_levels = &[_]std.log.ScopeLevel{
-        .{ .scope = .zml_module, .level = if (show_mlir) .debug else .warn },
+        .{ .scope = .@"zml/module", .level = if (show_mlir) .debug else .warn },
```

Comment:
you don't need such complicated log levels anymore, just .info for the whole thing should be enough

Reply:
ok